tests/phpunit/tests/kses.php

```diff
@@ -7,6 +7,51 @@
  */
 class Tests_Kses extends WP_UnitTestCase {
 
+	/**
+	 * Test callback for `wp_kses_normalize_entities()` regular expression.
+	 *
+	 * @dataProvider  data_wp_kses_xml_named_entities
+	 *
+	 * @ticket 54060
```

Comment:
```suggestion
	 * @dataProvider  data_wp_kses_xml_named_entities
	 * @covers ::wp_kses_xml_named_entities
	 * @ticket 54060
```

Reply:
Hi @peterwilsoncc, thanks for the suggestion, annotation is added now.